lib/web_ui/lib/src/engine/platform_dispatcher/view_focus_binding.dart

```diff
@@ -69,8 +69,7 @@ final class ViewFocusBinding {
   });
 
   late final DomEventListener _handleKeyDown = createDomEventListener((DomEvent event) {
-    event as DomKeyboardEvent;
-    if (event.shiftKey ?? false) {
+    if (event is DomKeyboardEvent && event.shiftKey == true) {
```

Comment:
Instead of `is`, I think this needs to be:

```dart
domInstanceOfString(event, 'DomKeyboardEvent')
```

So it also works in Wasm? There's a note about not using "is" (or "as") in the dart:js_interop package.

Reply:
Seems to be related to `JsAny` [only]. Who can we ask about this? 

[1]: https://dart.dev/interop/js-interop/js-types#compatibility-type-checks-and-casts

Reply:
@srujzs is the expert here.

Reply:
It's not specific to `JSAny` only - it affects any interop type declared with `dart:js_interop`. The documentation may be confusing there in the code samples. Use `isA` to test types:

```dart
if (event.isA<DomKeyboardEvent>() && ((event as DomKeyboardEvent).shiftKey ?? false)) {
```

Note that `isA` can't promote so you need the extra cast. You could also use `instanceofString` or any other relevant interop helpers, but that's probably more code than needed.